modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/js/DLManagementToolbarPropsTransformer.js

```diff
@@ -220,6 +220,18 @@ export default function propsTransformer({
 		});
 	};
 
+	const filterByTag = (tagsFilterURL) => {
+		openSelectionModal({
+			buttonAddLabel: Liferay.Language.get('select'),
+			height: '70vh',
+			multiple: true,
+			selectEventName: `${portletNamespace}selectedAssetTag`,
+			size: 'lg',
```

Comment:
I would say that we should use the same modal size with than in filter by extension (`size: 'md'`)

Reply:
I tried but it look like 
<img width="862" alt="image" src="https://user-images.githubusercontent.com/47524751/233417495-17188b00-851d-451e-b711-ecd619a07800.png">

the search bar is missing.

Reply:
Ok, after seeing this with you via slack, I agree you should keep the size='lg', as content-dashboard does the same, and we cannot change the internal view.

Reply:
thanks!